java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -77,7 +74,6 @@ disabled.modules=\
     org.netbeans.libs.git,\
     org.netbeans.libs.graaljs,\
     org.netbeans.libs.ini4j,\
-    org.netbeans.libs.jaxb,\
```

Comment:
Please doublecheck the dependencies. I can see some of them starting recursively from `cloud.common` (but there's no impl of cloud.common SPI ?) but which of the 'feature' modules needs bcprov jaxb or restlib ? Isn't there some impl-providing module missing ?

Reply:
This is really needed: `Warning - could not install some modules:
	org.netbeans.modules.xml.jaxb.api - No module providing the capability com.sun.xml.bind could be found.
	org.netbeans.modules.websvc.restlib - The module org.netbeans.modules.xml.jaxb.api would also need to be installed.
	org.netbeans.modules.cloud.oracle - The module org.netbeans.modules.websvc.restlib would also need to be installed.`

Reply:
Aha ! So this fixes an earlier addition of `cloud.oracle` (since it was added / not disabled in NBLS in #3352) ... Thanks.